projects/workshop/rickAndMortyApi/README.md

```
@@ -0,0 +1,32 @@
+# Rick and Morty
+
+Esse é um aplicativo feito em Flutter que usa a Rick and Morty API para mostrar uma lista de personagens do desenho.
+Cada personagem aparece num card com a imagem e o nome.
+O Aplicativo possui uma barra de busca que filtra os personagens enquanto você digita e, se clicar em um deles, ira para outra página para ver mais detalhes.
+
+
+## Arquitetura do Projeto
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)